src/sentry/sentry_metrics/indexer/cache.py

```diff
@@ -25,7 +32,7 @@ def randomized_ttl(self) -> int:
 
     def make_cache_key(self, key: str, cache_namespace: str) -> str:
         hashed = md5_text(key).hexdigest()
-        return f"indexer:org:str:{cache_namespace}:{hashed}"
+        return f"indexer{self.partition_key}:org:str:{cache_namespace}:{hashed}"
```

Comment:
Should this be:
```suggestion
        return f"indexer:{self.partition_key}:org:str:{cache_namespace}:{hashed}"
```

Reply:
@lynnagara yeah I first had that but wanted to avoid `indexer::org...` but also I thought maybe it would be easier if pattern of the key was the same in that you didn't have some that had `indexer:cs:org` and some that were just `indexer:org`, so that if you were to split on `":"` you'd know what to expect.